Scripts/createbinaries.bat

```diff
@@ -22,8 +22,8 @@ if %version%.==. goto :usage
 @rem ==============
 
 
-@rem Moving build to VS2017
-@set msbuildpath="%ProgramFiles(x86)%\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin\MSBuild.exe"
+@rem Moving build to VS2019
+@set msbuildpath="%ProgramFiles(x86)%\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\amd64\MSBuild.exe"
 @if not exist %msbuildpath% set @set msbuildpath="%ProgramFiles(x86)%\MSBuild\12.0\Bin\MSBuild.exe"
 @if not exist %msbuildpath% set msbuildpath=%windir%\microsoft.net\framework\v4.0.30319\msbuild.exe

```

Comment:
I did this hackiness before `dotnet.exe` was around. You could try to see if `set msbuildexe=dotnet.exe build` works.